libs/components/indicators/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -30,12 +30,95 @@ describe('Help inline component', () => {
       .query(By.css('.sky-help-inline'))
       .triggerEventHandler('click', undefined);
     fixture.detectChanges();
-    expect(cmp.buttonIsClicked).toBe(true);
+    expect(cmp.showHelpText).toBe(true);
   });
 
-  it('should pass accessibility', async () => {
+  it('should pass accessibility (ariaControls: undefined, ariaExpanded: undefined)', async () => {
     fixture.detectChanges();
     await fixture.whenStable();
+    expect(
```

Comment:
These tests all have similar contents, just with what's expected changing between each. Factor out into a function that accepts the expected values (use `null` with `toBe(null)` when parameterizing the new function). Also factor out the call to `fixture.nativeElement.querySelector('.sky-help-inline')` so you're not calling it repeatedly.

Reply:
Done